Python/pywarpx/fields.py

```diff
@@ -140,6 +140,23 @@ def __getitem__(self, index):
         elif self.dim == 3:
             return self._getitem3d(index)
 
+    def _find_start_stop(self, ii, nn, ngrow, d):
+        if isinstance(ii, slice):
```

Comment:
Just a quick idea, can we add a parameter description for this helper function? :)
```suggestion
        """Short description

        ii : integer
            ...
        nn : integer
            ...
        ngrow : integer
            ...
        d : integer
            ...
        """
        if isinstance(ii, slice):
```